jenkins/pipelines/cd/tiup/grafana-tiup-mirror-update.groovy

```diff
@@ -0,0 +1,109 @@
+def fallback=false
```

Comment:
Why need this var to control the flow?

Reply:
we have not cherry-pick script pr to all the branch, so we need this to fallback to old pipeline